<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>656</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>주식 자동매매</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_password">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호 입력</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_account">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_market">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	padding:5px
}

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>76</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>23</number>
    </attribute>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>코스피</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>코스닥</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>코스피200</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>59</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>25%씩 - 2% / 4% / 6% / 8%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50%씩 - 3%  / 6% </string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sell_btn">
    <property name="geometry">
     <rect>
      <x>769</x>
      <y>60</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>분할매도</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_chegul">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>451</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>451</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>체결내역</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>로그창</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="stocksTable">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	padding: 5px
}
QTableWidget::item {
	setTextalign: center
}
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>종목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>보유수량</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매수가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평가손익</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수익률(%)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>256</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>코드</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>종목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>조건명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>삭제</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="orderBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>주문</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>34</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="orderTypeComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>신규매수</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>신규매도</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>매수취소</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>매도취소</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="priceSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>149</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="accountComboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>69</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>종목</string>
    </property>
    <property name="buddy">
     <cstring>codeLineEdit</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="hogaTypeComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>88</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>지정가</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>시장가</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>주문</string>
    </property>
    <property name="buddy">
     <cstring>orderTypeComboBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>가격</string>
    </property>
    <property name="buddy">
     <cstring>priceSpinBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>종류</string>
    </property>
    <property name="buddy">
     <cstring>hogaTypeComboBox</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeNameLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
    <property name="buddy">
     <cstring>accountComboBox</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="qtySpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>119</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>수량</string>
    </property>
    <property name="buddy">
     <cstring>qtySpinBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>51</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 잔액</string>
    </property>
    <property name="buddy">
     <cstring>accountComboBox</cstring>
    </property>
   </widget>
   <widget class="QTableWidget" name="hoga_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>24</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>잔량</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매도호가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매수호가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>잔량</string>
     </property>
    </column>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="inquiryBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>46</height>
     </size>
    </property>
    <property name="text">
     <string>계좌 조회</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="realtimeCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>실시간 조회</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>조건식</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_condition">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="condition_btn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>매수 등록</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>292</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>매수 조건</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>매수 후 예약매도</string>
    </property>
   </widget>
   <widget class="QComboBox" name="buyamount">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>290</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>100,000 원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200,000 원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300,000 원</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>금액</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>단순 : 매수주문 수량의</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>수익률</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>복합 옵션</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>25% 씩 - 2% / 4% / 6% / 8%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50% 씩 - 2% / 4%</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>현재가</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>매수가</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>대비</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_buycond">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>매수호가 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>매수호가 2</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
